src/vec3_linalgebra.c

```diff
@@ -19,7 +19,11 @@ float	ft_vec3_abs(t_vec3 v)
 
 t_vec3	ft_vec3_norm(t_vec3 v)
 {
-	return (ft_vec3_scale(v, 1.0 / ft_vec3_abs(v)));
+	const float	len = ft_vec3_abs(v);
+
+	if (len == 0.0)
```

Comment:
Is there any occurence of null vectors possible in the program runtime after implementing the check for it which aborts if one is given? If none can be introduced thereafter this check is superfluous

Reply:
I don't know. We also wouldn't immediately notice it, since its a float. This means div zero doesn't crash but would return infinity or NaN (standard float implementation has bits reserved for that)

In #23  we decided to add it. Call also remove it again.

Reply:
Agreed to remove that length check for performance gains